### ISSUE TITLE
Releax the Rubocop::Cop::TrivialAccessors

### ISSUE THREAD

**Issue by famished-tiger** [rubocop/rubocop#211]
This cop is a bit too strict.  
Consider the following class:  

``` ruby
class Foo
  attr_reader(:bar)
  def initialize(something)
    @bar = something
...
```

The cop complains about the initialization of the attribute bar: `"Use attr_writer to define trivial writer methods."`
The issue is that I want bar to have a public read-only accessor and no writer (this attribute is initialized once only in the constructor).
Following the advice from the cop, will oblige:
-Create a writer it should be kept private method.
-That simple writer will only be used in the constructor. Thus a writer method for one single calling method!...
This will add burden to the class.
In fact, the cop should be excluded for any attribute initialized in the constructor and that attribute is nowhere else updated. Easy to say, but I realize that's a challenging check to implement.
